package.json

```diff
@@ -41,7 +41,13 @@
     "sentry-upload-sourcemaps": "sentry-cli --auth-token $SENTRY_AUTH_TOKEN releases -o $SENTRY_ORG -p $SENTRY_PROJECT files $npm_package_version upload-sourcemaps ./build/static/js/",
     "prepare": "husky install",
     "cypress:open": "cypress open",
-    "cypress:run": "cypress run"
+    "cypress:open:dev": "cypress open --config baseUrl=https://safe-team.dev.gnosisdev.com/app",
+    "cypress:open:staging": "cypress open --config baseUrl=https://safe-team.staging.gnosisdev.com/app",
+    "cypress:open:prod": "cypress open --config baseUrl=https://gnosis-safe.io/app",
+    "cypress:run": "cypress run",
+    "cypress:run:dev": "cypress run --config baseUrl=https://safe-team.dev.gnosisdev.com/app",
+    "cypress:run:staging": "cypress run --config baseUrl=https://safe-team.staging.gnosisdev.com/app",
+    "cypress:run:prod": "cypress run --config baseUrl=https://gnosis-safe.io/app"
```

Comment:
I would like to explore if there's other way to pass the environment other than writing a script for each one of the environments. Can we talk about it?

Reply:
I'll look into the following suggestion on Friday. Remove the above commands and use `cypress:run`/`:open` to then ask the following in the CL:

`"Which environment do you want to test? (l)ocal, (d)evelopment, (s)taging or (p)roduction?"`

What do you think?

Reply:
I've added a prompt that allows the user to pick the `baseUrl`. What do you think?